package.json

```diff
@@ -18,41 +18,43 @@
   },
   "private": true,
   "dependencies": {
-    "@nx/devkit": "17.2.8",
-    "@swc/helpers": "~0.5.2",
+    "@nx/devkit": "19.8.4",
+    "@swc/helpers": "0.5.13",
     "tslib": "^2.3.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^18.6.0",
-    "@commitlint/config-conventional": "^18.6.0",
-    "@jscutlery/semver": "^4.2.0",
-    "@nx/eslint": "17.2.8",
-    "@nx/eslint-plugin": "17.2.8",
-    "@nx/jest": "17.2.8",
-    "@nx/js": "17.2.8",
-    "@nx/plugin": "17.2.8",
-    "@nx/workspace": "17.2.8",
-    "@swc-node/register": "~1.6.7",
-    "@swc/cli": "~0.1.62",
-    "@swc/core": "~1.3.85",
-    "@types/jest": "^29.4.0",
-    "@types/node": "18.16.9",
-    "@typescript-eslint/eslint-plugin": "^6.9.1",
-    "@typescript-eslint/parser": "^6.9.1",
-    "eslint": "~8.48.0",
-    "eslint-config-prettier": "^9.0.0",
-    "husky": "^9.0.10",
-    "jest": "^29.4.1",
-    "jest-environment-jsdom": "^29.4.1",
-    "ngx-deploy-npm": "^8.0.1",
-    "nx": "17.2.8",
-    "prettier": "^2.6.2",
-    "ts-jest": "^29.1.0",
-    "ts-node": "10.9.1",
-    "typescript": "~5.2.2",
-    "verdaccio": "^5.0.4"
+    "@commitlint/cli": "^18.6.1",
+    "@commitlint/config-conventional": "^18.6.3",
+    "@eslint/eslintrc": "^2.1.4",
+    "@jscutlery/semver": "^5.3.1",
+    "@nx/eslint": "19.8.4",
+    "@nx/eslint-plugin": "19.8.4",
+    "@nx/jest": "19.8.4",
+    "@nx/js": "19.8.4",
+    "@nx/plugin": "19.8.4",
+    "@nx/workspace": "19.8.4",
+    "@stylistic/eslint-plugin": "^2.9.0",
```

Comment:
Why this package has been added? We are using `prettier` which works really well and should be sufficient.
Is this an addition made by Nx migrate? 😉

Reply:
There's a style rule `no-extra-semi` which you'll see in the eslint config this has also been migrated to the new flat config file this moved from `@typescript-eslint/eslint-plugin` to `@stylistic/eslint-plugin`

Reply:
This rule is not in the actual ESLint config, did Nx add it when it upgraded to v19?
https://github.com/nx-go/nx-go/blob/1715cc75736c7014096128c3d80f4af2836e8069/.eslintrc.json#L34-L38

Reply:
Sorry my mistake, yes, the upgrade to Nx did this:

https://github.com/nx-go/nx-go/pull/137/commits/6bb2ca0738b05a070a9d2d85b9c0cedd14e495b6